### ISSUE TITLE
Error when calling lovernet:getData() on server 

### ISSUE THREAD

**Issue by Sheepolution** [josefnpat/LoverNet#11]
`return self._cache[name] or self._ops[name].default_server(self)`

You use self._cache but in the .new() you give ._cache only to the client

```
if self._type == lovernet.mode.client then
    self._host = self._enet.host_create()
    self._host:connect(self._ip .. ':' .. self._port)
    self._cache = {}
  else --if self._type == lovernet.mode.server then
    self._host = self._enet.host_create("*:"..self._port)
  end
```

**Comment by josefnpat** [comment]
The `:getData()` function is only for the client - is there a reason you would want it on the server? Can't you use the storage object that is on the server that's passed in all the `addProcessOnServer()` functions?

Relates to #5 as `:getData()` shouldn't be available to the server.

**Comment by Sheepolution** [comment]
Oh no that's just my lack of understanding how this lib works. I just wanted to point out that this didn't make sense, and that either getData shouldn't be available on server or cache should be available on server. I wasn't aware of #5

**Comment by josefnpat** [comment]
Fixed in cleanup of api: eaab962c1c9d43ec21c444f8fc40ec4371da93d3

Check out the new ldocs! <3

**Comment by josefnpat** [comment]
Specifically: 9b49ecac51a4334646ac7d7314ddb8cb04580647
